hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/steps/StakePeriodChanges.java

```diff
@@ -110,12 +109,6 @@ public boolean process(
             }
             final var config = tokenContext.configuration();
             try {
-                final var nodeStore = newWritableNodeStore(stack, config);
-                final BiConsumer<Long, Integer> weightUpdates = (nodeId, weight) -> nodeStore.put(nodeStore
```

Comment:
This BiConsumer used to be an arg in `updateNodes()` call below, but has been removed there recently and isn't used any longer.